atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/AbstractTransactionManager.java

```diff
@@ -53,7 +53,7 @@
                 if (!e.canTransactionBeRetried()) {
                     log.warn("[{}] Non-retriable exception while processing transaction.",
                             SafeArg.of("runId", runId),
-                            SafeArg.of("failureCount", failureCount));
+                            e);
```

Comment:
Checking this codepath, it seems that the only relevant code that hits it is the Background Sweep (apart from CLIs or unused code), which also logs the stacktrace. If this is the case, we may not want to log the stacktrace. However, since AtlasDB is a library, this exception may not be caught by all our consumers.

Also, only the one on line 66 seems to be a somewhat hot codepath, being logged every time a transaction fails. But even this one doesn't seem to trigger very often. So I'd be more inclined to not change the previous behavior.